### ISSUE TITLE
Settings Editor: Parent Null error when keyboard navigating settings

### ISSUE THREAD

**Issue by cleidigh** [Microsoft/vscode#55509]
Insiders master 8/1/18
Windows

Error Cannot read parent of 'null' when navigating the settings editor with keyboard

Steps to reproduce:

Open new settings
starts in search bar
navigate down with keyboard to files autosave delay input box
navigate back up the list 
error usually occurs just before header

![image](https://user-images.githubusercontent.com/25272315/43483823-ad971a24-94da-11e8-9ab6-77d0e656a7cc.png)

**Comment by roblourens** [comment]
Thanks!

**Comment by cleidigh** [comment]
FYI
I think you can get this with a variety of Tabs and arrow keys as well
